<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Store System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>List Items</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-20</y>
          <width>421</width>
          <height>301</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Filter Search</string>
           </property>
           <widget class="QRadioButton" name="allButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>All</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="poetryButton">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>30</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Poetry</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="novelButton">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>30</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Novel</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="scienceButton">
            <property name="geometry">
             <rect>
              <x>320</x>
              <y>30</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Science</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>80</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Search Book</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Writer:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>140</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Book Name:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>170</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Publisher:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="writer_input">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>110</y>
              <width>321</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="bookname_input">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>140</y>
              <width>321</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="publisher_input">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>170</y>
              <width>321</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="searchButton">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>210</y>
              <width>421</width>
              <height>32</height>
             </rect>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>0</y>
          <width>481</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>477</width>
              <height>277</height>
             </rect>
            </property>
            <widget class="QTableWidget" name="tableWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>481</width>
               <height>281</height>
              </rect>
             </property>
             <property name="rowCount">
              <number>0</number>
             </property>
             <column>
              <property name="text">
               <string>Book</string>
              </property>
              <property name="textAlignment">
               <set>AlignLeading|AlignVCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Writer</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Kind</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Publisher</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Stock</string>
              </property>
             </column>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>911</width>
          <height>331</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.SF NS Text'; font-size:18pt; font-weight:600;&quot;&gt;Book Store System Application v1.0&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.SF NS Text'; font-size:14pt; font-weight:600;&quot;&gt;Developed by: Ula≈ü Eraslan&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.SF NS Text'; font-style:italic;&quot;&gt;Contact: ulaseraslan@sabanciuniv.edu&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.SF NS Text'; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.SF NS Text'; font-size:18pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
